spark/src/test/java/org/apache/iceberg/spark/data/parquet/vectorized/TestParquetDictionaryEncodedVectorizedReads.java

```diff
@@ -38,7 +39,8 @@
 public class TestParquetDictionaryEncodedVectorizedReads extends TestParquetVectorizedReads {
 
   @Override
-  Iterable<GenericData.Record> generateData(Schema schema, int numRecords, long seed, float nullPercentage) {
+  Iterable<GenericData.Record> generateData(Schema schema, int numRecords, long seed, float nullPercentage,
+                                            Function<GenericData.Record, GenericData.Record> transform) {
```

Comment:
It's strange that this adds the transform and then ignores it. Is there a better way to structure this so that it doesn't require doing this? Maybe just don't call this method in a subclass?